.github/workflows/ci-llama-large-tests.yaml

```diff
@@ -76,7 +77,7 @@ jobs:
             iree-base-runtime
 
       - name: Run llama tests
-        run: pytest sharktank/tests/models/llama/benchmark_amdgpu_test.py -v -s --run-nightly-llama-tests --iree-hip-target=gfx942 --html=out/llm/llama/benchmark/index.html
+        run: pytest sharktank/tests/models/llama/benchmark_amdgpu_test.py -v -s --run-nightly-llama-tests --iree-hip-target=gfx942 --iree-device=7 --html=out/llm/llama/benchmark/index.html
```

Comment:
Ideally [--iree-device](https://github.com/nod-ai/shark-ai/blob/main/sharktank/conftest.py#L176) takes the form `hip://<id>`. You would want to alter `--iree-device` str accordingly in [iree_benchmark_vmfb](https://github.com/nod-ai/shark-ai/blob/main/sharktank/sharktank/utils/export_artifacts.py#L275).